gpu-aware-scheduling/docs/usage.md

```diff
@@ -4,34 +4,34 @@ This document explains how to get GAS working together with [Node Feature Discov
 To begin with, it will help a lot if you have been successful already using the GPU-plugin with some deployments. That means your HW and cluster is most likely fine with GAS also.
 
 ## GPU-plugin
-Resource management enabled version of the GPU-plugin is currently necessary for running GAS. The resource management enabled GPU-plugin version can read the necessary annotations of the PODs, and without those GPU allocations will not work correctly. It can be deployed directly via kustomized yamls by issuing:
-````
-kubectl apply -k deployments/gpu_plugin/overlays/fractional_resources
-````
+Resource management enabled version of the GPU-plugin is currently necessary for running GAS. The resource management enabled GPU-plugin version can read the necessary annotations of the PODs, and without those GPU allocations will not work correctly. A copy of the plugin deployment kustomization can be found from [docs/gpu_plugin](./gpu_plugin). It can be deployed simply by issuing:
```

Comment:
"and without those GPU allocations..." would mean "and without those annotations, GPU allocations..." ?

Reply:
yes, fixed